analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -938,7 +938,8 @@ def parse_unique_log(compilation_database,
                      compiler_info_file=None,
                      keep_gcc_fix_headers=False,
                      analysis_skip_handler=None,
-                     pre_analysis_skip_handler=None):
+                     pre_analysis_skip_handler=None,
+                     ctu_or_stats_enabled=False):
```

Comment:
Couldn't this information be included in `pre_analysis_skip_handler` object? This way we could save this extra parameter, because this function has already too many parameters. `should_skip()` function of the handler could return the appropriate value depending on whether ctu or stats was enabled.

Reply:
Ok, I approve this PR, but I still think that all the skip handler related issues should be handled in one class.